railties/test/application/rake/dbs_test.rb

```diff
@@ -118,23 +109,21 @@ def db_create_with_warning(expected_database)
         db_create_and_drop("db/development.sqlite3", environment_loaded: false)
       end
 
-      test "db:create and db:drop show warning but doesn't raise errors when loading YAML with alias ERB" do
+      test "db:create and db:drop don't raise errors when loading ERB accessing nested configurations" do
         app_file "config/database.yml", <<-YAML
-          sqlite: &sqlite
-            adapter: sqlite3
-            database: db/development.sqlite3
-
           development:
-            <<: *<%= ENV["DB"] || "sqlite" %>
+            database: db/development.sqlite3
```

Comment:
This is no longer testing the use case that it was describing, using ERB as a YAML alias. Can you put the test back, or add a second one so that we are still testing that case?

Reply:
I don't know why I removed this test initially. I put it back, slightly altered, to show, that this now works as well.